_pages/policies/travel-and-leave-policies/first-time-travel-travel-card.md

```diff
@@ -24,26 +24,34 @@ _Est. time: 60 minutes_
 6. Once completed there is a second training to take. Return to the "Find Learning" search box and search for "travel card".
 7. Select "GSA Travel Card Training", and then Start course.
 8. Once completed, download your certificates of completion for both trainings by returning to the "Learning" page and clicking on the "View All" button under "Learning History". You should see your completed travel card and TDY trainings on the list, and there will be an icon to generate a print preview, which you can save, at the right.
-9. For ease of processing later on, save your certificates as “Your Name travel card training” and “Your Name TDY training”.
+
+***NOTE***: Save your certificates as `FIRSTNAME_LASTNAME-TravelCardTrainingCert.pdf` and `FIRSTNAME_LASTNAME-TDYTrainingCert.pdf` for ease of processing later on.
 
 ### 2. Fill out the Supervisory Travel Card Approval
 
 _Estimated time: 10 minutes plus wait time for your supervisor’s signature_
 
 1. Download the [Supervisory Travel Card Approval Form](https://insite.gsa.gov/cdnstatic/Supervisory%20Travel%20Card%20Approval%20Form%20-%20GSA7662-19.pdf) and fill out the following fields.
 
-  * **Employee name:** Your name
+  * **Employee Name:** Your name
```

Comment:
May want to double-check with #tts-handbook, but I'm pretty sure we prefer sentence case.

Reply:
I'm only using this particular case because that's how it appears on the form itself.